test/melange-ppx-private.t

```diff
@@ -0,0 +1,29 @@
+Test to showcase errors when using %%private extensions with melange ppx
+
+  $ . ./setup.sh
+  $ cat > dune-project <<EOF
+  > (lang dune 3.7)
+  > (using melange 0.1)
+  > EOF
+  $ cat > dune <<EOF
+  > (melange.emit
+  >  (target output)
+  >  (alias mel)
+  >  (emit_stdlib false)
+  >  ;(preprocess (pps melange.ppx))
```

Comment:
I don't get this change. If the ppx is disabled, then we don't see the error, right?

Reply:
direct link to diff: https://github.com/melange-re/melange/commit/d87e473823762fc0901efac86594bd196e477dec#diff-cf3151c38c0d5e126958e640fd71862a0d3ecf6ec26f7ce935b1d3028062772d

Reply:
I see, thanks!